### PR TITLE
Mandatory params v1.2

### PR DESCRIPTION
**Pull request by catalinnnn**

### CODE REVIEW

File: dex/farm/src/migration.rs [hunk 1]
```diff
@@ -96,7 +96,7 @@ pub trait MigrationModule:
 
         Ok(self
             .farm_v1_4_contract_proxy(new_farm_dest)
-            .migrate_from_v1_2_farm(orig_caller_opt)
+            .migrate_from_v1_2_farm(orig_caller)
```
Comment (camilbancioiu) [inline dex/farm/src/migration.rs:99]: Wouldn't `self.blockchain().get_caller()` work here? It doesn't need to be an argument to the endpoint.

Comment (catalinnnn) [reply dex/farm/src/migration.rs:99]: User calls Old Farm that calls New Farm. New Farm should send the tokens to User

Comment (catalinnnn) [reply dex/farm/src/migration.rs:99]: Also, User can Call Proxy that Calls Old Farm that calls New Farm. New Farm should send the tokens to User

Comment (catalinnnn) [reply dex/farm/src/migration.rs:99]: So an Orig Caller is needed

**Review by camilbancioiu — APPROVED** [review]

**Review by sasurobert — APPROVED** [review]

**Pull request merged**